packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -291,25 +301,21 @@ A custom email address allows you to present a branded experience to your end us
 
 Okta sends your super admins a confirmation email after your custom email address is configured and operating correctly. To ensure continuous operation, Okta polls your custom email domain once every 24 hours. If a problem occurs, Okta alerts super admins by email, and Okta-generated emails are sent from the default address `noreply@okta.com` until the problem is resolved.
 
-## Configure a custom email address
-
-1. In the Admin Console, go to **Customizations** > **Emails**.
+> **Note:** If you've enabled [Early Access (EA) multibrand customization](https://help.okta.com/okta_help.htm?type=oie&id=csh-branding), your Admin Console navigation is different. See parenthetical notes.
 
-2. Click the **Sender:** link (**Okta <noreply@okta.com>**) near the top of the page.
-
-3. On the Configure Email Sender dialog box, select **Custom email domain** as the type of sender that you want to send system notification emails from.
+## Configure a custom email address
 
-4. In the **Email address to send from**, enter the email address that you want to send the system notification emails from. This is what displays in the emails sent to your users.
+1. In the Admin Console, go to **Customizations** > **Emails**. (EA users: go to **Customizations** > **Brands**, and then select the brand you want. In the **Domains** tab, in the **Email domain** section, click **Add domain**).
```

Comment:
in the Email section, click Edit, then Add custom email domain

Reply:
done